pytest_selenium/pytest_selenium.py

```diff
@@ -303,6 +303,12 @@ def pytest_addoption(parser):
                   help='debug to exclude from capture',
                   default=os.getenv('SELENIUM_EXCLUDE_DEBUG'))
 
+    _auth_choices = ('none', 'token', 'hour', 'day')
+    parser.addini('saucelabs_job_auth',
+                  help='Authorization for the SauceLabs job {0}'.
+                  format(_auth_choices),
+                  default=os.getenv('SAUCELABS_JOB_AUTH', 'token'))
```

Comment:
I feel that the default should be NONE, for two reasons. The first is that this means there is no change of behaviour when upgrading, and the second is that I don't feel that we should be generating permanent public links to jobs by default.